src/Sylius/Bundle/AddressingBundle/Form/EventListener/BuildAddressFormSubscriber.php

```diff
@@ -137,6 +137,8 @@ private function createProvinceCodeChoiceForm(CountryInterface $country, $provin
         return $this->formFactory->createNamed('provinceCode', ProvinceCodeChoiceType::class, $provinceCode, [
             'country' => $country,
             'auto_initialize' => false,
+            'label' => 'sylius.form.address.province',
```

Comment:
When I define sylius.form.address.province to 'County' in a messages.yml, it will populate the ajax loaded province input, but the non-ajax loaded input still is labelled as 'Province Name'

Are you saying there's somewhere else I should be defining the labels instead of in the translation files?

The only reason I put this PR in is because these are the options on the inputs when loaded via ajax.

Perhaps it isn't needed for the province dropdown, my issue was only with the text input so it may be the problem is best fixed elsewhere.